apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/ResourceCard.js

```diff
@@ -1,15 +1,19 @@
-import Image from "next/image";
-import Button from "@devlaunchers/components/src/components/atoms/Button";
-import { ResourceCardContainer } from "./StyledResourceCard";
-
-function ResourceCard({resource : {img, title, guideTitle,guideLink}}) {
-    return (
-        <ResourceCardContainer>
-            <Image src={`/images/projects/resources/logos/${img}`} width={50} height={50} alt={title}/>
-            <Button buttonSize="standard" buttonType="primary">{title}</Button>
-            <a href={guideLink} >guide to {guideTitle}</a>
-        </ResourceCardContainer>
-    );
-}
-
-export default ResourceCard;
\ No newline at end of file
+
+import { ResourcersCards, ResourcersIcon, CardsBtn, ResourcersIconInfo } from "./StyledResourceCard";
+
+function ResourceCard({resource : {img, title, guideTitle,guideLink, moreInfo}}) {
+    return (
+           <ResourcersCards>
+            <ResourcersIcon src={`/images/projects/resources/logos/${img}`}  alt={title}/>
+            <a href={guideLink} target="_blank">
+            <CardsBtn>{title}</CardsBtn>
```

Comment:
Hey Hebertt, great work everything is in order, just one thing we need to use the UI elements from UI packages so we have a universal design on the website, so you can do : 

```
import Button from "@devlaunchers/components/src/components/atoms/Button";

<Button buttonSize="standard" buttonType="secondary">{title}</Button>
```
i merged `developement/new-builds` branch containing the updated theme into this branch so you can use those UI components.